src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -223,11 +223,33 @@ public static boolean onDrawHighlight(LevelRenderer context, Camera camera, HitR
         }
     }
 
+    @Deprecated(forRemoval = true, since = "1.18.2")
     public static void dispatchRenderLast(LevelRenderer context, PoseStack poseStack, float partialTick, Matrix4f projectionMatrix, long finishTimeNano)
     {
         MinecraftForge.EVENT_BUS.post(new RenderLevelLastEvent(context, poseStack, partialTick, projectionMatrix, finishTimeNano));
     }
 
+    /**
+     * The current partial ticks when LevelRenderer.renderLevel is called.
+     * Use this for places where partial ticks are unavailable, but do not modify it.
+     */
+    public static float partialTicks = 0.0F;
```

Comment:
I'd prefer this be named `partialTick` to be consistent with Mojang's naming of `net.minecraft.client.Timer#partialTick` (which is the root of the partial tick values during rendering) and the naming of the field in the event.